src/generated/cloud/aiplatform/v1/Cargo.toml

```diff
@@ -26,6 +26,74 @@ keywords.workspace     = true
 categories.workspace   = true
 rust-version.workspace = true
 
+[features]
+default = [
```

Comment:
Q: should we be opinionated about the default set?

e.g. if our telemetry indicated that 99.9% of requests were for either `prediction_service` or `gen_ai_tuning_service`, we'd probably prefer for those two features to be the extent of our default set.

We are allowed to add default features, but not allowed to remove default features.

(Dont let this block your PR though. Just something to think about before v1.0)